### PR TITLE
Rwa 313

### PR DESCRIPTION
**Pull request by chrisdavecm**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-313

### Change description ###
Add secondary sorting

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dharmendrak — DISMISSED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java [hunk 1]
```diff
@@ -1022,10 +1025,10 @@ private static Stream<TaskQueryScenario> withAllGrantTypesHappyPath() {
                     "8d6cc5cf-c973-11eb-bdba-0242ac111024", "1623278362431024",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111003", "1623278362431003",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111023", "1623278362431023",
-                    "8d6cc5cf-c973-11eb-bdba-0242ac111006", "1623278362431006",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111007", "1623278362431007",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111008", "1623278362431008",
-                    "8d6cc5cf-c973-11eb-bdba-0242ac111009", "1623278362431009"
+                    "8d6cc5cf-c973-11eb-bdba-0242ac111009", "1623278362431009",
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java:1030]: As we now have the AC's locked in for the sort, can we embellish taskDetail assertion here to include the dueDate. So it shows the date on the task as Desc.

File: src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java [hunk 2]
```diff
@@ -1248,6 +1251,97 @@ private static Stream<TaskQueryScenario> defaultSortedResultsScenario() {
...
+            .expectedTotalRecords(8)
+            .expectedTaskDetails(newArrayList(
+                "8d6cc5cf-c973-11eb-bdba-0242ac222005", "1623278362222005",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222000", "1623278362222000",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222001", "1623278362222001",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222002", "1623278362222002",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222003", "1623278362222003",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222004", "1623278362222004",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222006", "1623278362222006",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222007", "1623278362222007")
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java:1276]: can we include locationName to show ASC?

File: src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java [hunk 3]
```diff
@@ -1248,6 +1251,97 @@ private static Stream<TaskQueryScenario> defaultSortedResultsScenario() {
...
+            .expectedTaskDetails(newArrayList(
+                "8d6cc5cf-c973-11eb-bdba-0242ac222005", "1623278362222005",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222000", "1623278362222000",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222001", "1623278362222001",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222002", "1623278362222002",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222003", "1623278362222003",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222004", "1623278362222004",
+
+                "8d6cc5cf-c973-11eb-bdba-0242ac222006", "1623278362222006",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222007", "1623278362222007"
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java:1307]: can we include caseName to show ASC?

File: src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java [hunk 4]
```diff
@@ -1248,6 +1251,97 @@ private static Stream<TaskQueryScenario> defaultSortedResultsScenario() {
...
+            .expectedTotalRecords(8)
+            .expectedTaskDetails(newArrayList(
+                "8d6cc5cf-c973-11eb-bdba-0242ac222000", "1623278362222000",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222001", "1623278362222001",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222002", "1623278362222002",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222003", "1623278362222003",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222004", "1623278362222004",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222006", "1623278362222006",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222007", "1623278362222007",
+                "8d6cc5cf-c973-11eb-bdba-0242ac222005", "1623278362222005")
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java:1337]: can we include locationName to show DESC?

**Review by raghera — APPROVED** [review]

**Pull request merged**